Doc improvement (#1452)
ClusterLabs/crmsh: doc/crm.8.adoc

==== `location`

`location` defines the preference of nodes for the given

> resource. The location constraints consist of one rule
> which specify a score to be awarded if the rule matches.

Reviewer: ```suggestion
resource. The location constraints are defined by a single rule
which specifies a score to be awarded if the rule matches.
```
